src/katsdpnetboxutilities/system/README.md

```diff
@@ -0,0 +1,29 @@
+# Netbox device querying
```

Comment:
You have made things a lot better, thank you. Can you now update this Readme so that someone knows how to use it. 
I'd like to see and example on using the make file and an example with docker only, without the build and assuming the image is harbor.sdp.kat.ac.za/infra/sysinfo